File: Python/uri_1012.py
import math

a, b, c = input().split(' ')
a = float(a)
b = float(b)
c = float(c)
triangulo = (a * c) / 2.0
circulo = 3.14159 * math.pow(c, 2)
trapezio = ((a + b) / 2.0) * c
quadrado = b * b
retangulo = a * b
print('TRIANGULO: %0.3f' % triangulo)
print('CIRCULO: %0.3f' % circulo)
print('TRAPEZIO: %0.3f' % trapezio)
print('QUADRADO: %0.3f' % quadrado)
print('RETANGULO: %0.3f' % retangulo)

File: Go/uri_1012.go
package main

import "fmt"
import "math"

func main() {
	var a, b, c float64
	fmt.Scan(&a, &b, &c)
	fmt.Printf("TRIANGULO: %.3f\n"+
		"CIRCULO: %.3f\n"+
		"TRAPEZIO: %.3f\n"+
		"QUADRADO: %.3f\n"+
		"RETANGULO: %.3f\n", (a*c)/2.0, 3.14159*math.Pow(c, 2), (a+b)*c/2.0, b*b, a*b)
}
